src/util/basketUtils.js

```diff
@@ -42,6 +44,7 @@ export function setLendAmount({ amount, apollo, loanId }) {
 			optimisticResponse: {
 				__typename: 'Mutation',
 				shop: {
+					id: basketId,
```

Comment:
I think this should just be `0` as I don't believe it's the same as the basketId we use from the cookie.